ofono/doc/radio-settings-api.txt

```diff
@@ -44,6 +44,12 @@ Properties	string TechnologyPreference [readwrite]
 				"gsm"	Only GSM used for radio access.
 				"umts"	Only UMTS used for radio access.
 				"lte"	Only LTE used for radio access.
+				"umts,gsm"	Dual mode operation with UMTS
+					and GSM radio access whith preference
+					for UMTS.
+				"lte,umts"	Dual mode operation with LTE
+					and UMTS radio access with preference
+					for LTE.
```

Comment:
That's not going to work simply like that. "umts" is already interpreted by Sailfish OS as "umts,gsm", "lte" as LTE and anything below, and it's going to stay that way. We need a different syntax for "lte only" and "umts only" modes, to keep D-Bus interface backward compatible.

I've been thinking about it couple years ago: https://github.com/monich/ofono/commits/rats

👆It's not a finished work, I need to take another look at it before I can say if it's still a valid approach.

Reply:
Ok, let me know if there is a commit (or several) from the PR that need to be dropped.

Reply:
@monich Have you had time to look into the branch yet?

Reply:
Unless there are good enough reasons otherwise, I'm thinking it could be better if we just switched the sailfish interpretation instead of indefinitely keeping a behavior difference to upstream.

Reply:
I was thinking backward compatibility (with 3rd party software which should survive upgrades) but since no one seems to care about it anymore then I can't think of any other reason🤷‍♂️